network-store-iidm-impl/src/main/java/com/powsybl/network/store/iidm/impl/NodeBreakerViewImpl.java

```diff
@@ -100,8 +100,11 @@ public int getBusbarSectionCount() {
     @Override
     public BusbarSection getBusbarSection(String id) {
         checkBusBreakerTopology();
-        return index.getBusbarSection(id)
-                .orElse(null);
+        BusbarSection bbs = index.getBusbarSection(id).orElse(null);
+        if (bbs != null && bbs.getTerminal().getVoltageLevel() != this.voltageLevel) {
```

Comment:
Maybe it would be safer to compare the voltage level id, instead of just comparing the object reference

Reply:
I used the same implementation as powsybl-core, should I still change this ? 
https://github.com/powsybl/powsybl-core/blob/main/iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/NodeBreakerVoltageLevel.java#L913

Reply:
in powsybl core, we can compare the object reference, because it's an implementation in memory
in network store, this is not the case

Reply:
fixed